airflow/providers/amazon/aws/operators/s3_list.py

```diff
@@ -72,8 +72,8 @@ def __init__(
         self,
         *,
         bucket: str,
-        prefix: str = '',
-        delimiter: str = '',
+        prefix: str,
+        delimiter: str,
```

Comment:
Is this change required?

Reply:
Actually no. I copied this code for the S3ListPrefixes operator - where you'd presumably want these to be explicit values, but that doesn't necessarily apply to S3List. Let me revert this and push it up. Good catch :)

Reply:
```suggestion
        prefix: str = '',
        delimiter: str = '',
```